ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/LegacyCliConfigConverter.scala

```diff
@@ -33,7 +33,7 @@ object LegacyCliConfigConverter {
         maxPayloadsPerPayloadsPage = cliConfig.eventsPageSize,
       ),
       bufferedStreamsPageSize = cliConfig.bufferedStreamsPageSize,
-      eventsProcessingParallelism = cliConfig.eventsProcessingParallelism,
+      bufferedEventsProcessingParallelism = cliConfig.eventsProcessingParallelism,
```

Comment:
Can you please rename the one in `cliConfig` as well?

Reply:
Ok, done